src/app/productConfig/productConfigModal/productConfig.modal.component.js

```diff
@@ -235,6 +235,19 @@ class ProductConfigModalController {
       this.submittingGift = false;
     } );
   }
+
+  displayName() {
+    return this.productData.displayName === this.itemConfig['jcr-title'] ?
+      this.productData.displayName : this.itemConfig['jcr-title'];
+  }
+
+  displayId() {
+    let value = `#${this.productData.designationNumber}`;
+    if(this.itemConfig['jcr-title'] && this.productData.displayName !== this.itemConfig['jcr-title']) {
```

Comment:
This condition doesn't make sense to me. Maybe `this.productData.displayName && this.productData.displayName !== this.itemConfig['jcr-title']`? If we have a display name and it isn't duplicate then we show it? The current condition seems like if the `jcr-title` is missing then we don't show this secondary title. Maybe that's what's intended...